services/subnet_manager/src/main/java/com/futurewei/alcor/subnet/service/implement/SubnetServiceImp.java

```diff
@@ -29,49 +30,86 @@
     private RestTemplate restTemplate = new RestTemplate();
 
     @Override
-    public void routeRollback(String routeId, String vpcId) {
-        String routeManagerServiceUrl = routeUrl + vpcId + "/routes/" + routeId; // for kubernetes test
+    public void routeFallback(String routeId, String vpcId) {
+        String routeManagerServiceUrl = routeUrl + "vpcs/" + vpcId + "/routes/" + routeId; // for kubernetes test
```

Comment:
Remove the comment if not applicable here.

Reply:
NP